content/spin/python-components.md

```diff
@@ -289,7 +289,7 @@ def handle_request(request):
     assert value == b"bar", f"expected \"bar\", got \"{str(value, 'utf-8')}\""
 
     return Response(200,
-                    [("content-type", "text/plain")],
+                    ("content-type", "text/plain"),
```

Comment:
Thanks for picking this up @sohanmaheshwar 

This will want `{` and `}` instead of parenthesis.
```suggestion
                    {"content-type": "text/plain"},
```
I have adjusted the code in one other area on this page, in relation to this also.
Nice catch, thank you!